docs/src/howto-guides/017_apama_software_management_plugin.md

```diff
@@ -1,70 +1,59 @@
 # Apama Software Management Plugin
 
-The apama plugin can be used to install and manage apama artefacts.
-This plugin can be used to install an apama project on a device as well as inject monitor files into an already installed project.
+The Apama plugin, part of the Apama thin-edge support package, can be used to install Apama projects using the Cumulocity software management feature.
 
+## Add the Apama thin-edge support package to the tenant software repository and deploy it to thin-edge
 
+1. In the **Device Management** app and go to the **Management** menu option and select the **Software repository**.
+2. Click **Add software** at the right of the top menu bar.
+3. In the **ADD SOFTWARE** dialog enter the following details:
+    - **Software**: apama-thin-edge-support
+    - **Description**: apama-thin-edge-support (or something else if you choose)
+    - **Device Filter Type**: (leave empty)
+    - **Software Type**: apt
+    - **Version**: 10.15.1.0-1::apt
```

Comment:
This is the version from the 10.15.1 branch - it should be 10.15.0.2 in the published version

Reply:
cc887df